store/store_test.go

```diff
@@ -64,6 +64,10 @@ type configTestSuite struct{}
 
 var _ = Suite(&configTestSuite{})
 
+// this is what snap.E("1*") looks like when decoded into an interface{}
+var iOneStar = map[string]interface{}{"read": []interface{}{0., 1.}, "write": []interface{}{1.}}
```

Comment:
This is a five star use of a variable holding one star epoch after a three star decoding.

Reply:
you forgot the transformation step through ℤ*